### ISSUE TITLE
Does not work anymore after firefox update to version 45

### ISSUE THREAD

**Issue by mowgli** [downthemall/anticontainer#115]
I tried to find any hint about that issue. But after upgrading firefox from v44 to v45, downthemall anticontainer stopped to work completely. If I load the settings I always get a empty list and no former working plugin is working. Also the checkmark in DTA dialog is gone.

So it looks for me as that anticontainer is not used by DTA at all anymore.

I updated today to version 1.5 but without any success. The DTA version is 2.0.18.

Is there anybody out there, having the same problem? Or is there any hint how to fix the issue?

**Comment by mowgli** [comment]
Some maybe related output from mozilla console:
```
[Exception... "Component returned failure code: 0x80520012 (NS_ERROR_FILE_NOT_FOUND) [nsIXPCComponents_Utils.import]"  nsresult: "0x80520012 (NS_ERROR_FILE_NOT_FOUND)"  location: "JS frame :: jar:file:///home/klaus/.mozilla/firefox/b0ujmphk.default/extensions/anticontainer@downthemall.net.xpi!/components/services.js :: <TOP_LEVEL> :: line 43"  data: no]
```

**Comment by version365** [comment]
I am using the v45 ESR and it works fine for me..
Have you tried installing both addons on a fresh install of FF or maybe diff profile?

**Comment by poke** [comment]
Please make sure you’re using the new DownThemAll! version 3 with a current version of AntiContainer. Then everything should work fine. If not, please report back.

**Comment by mowgli** [comment]
-----BEGIN PGP SIGNED MESSAGE-----
Hash: SHA512

Hi,

Am So den  9. Okt 2016 um 18:18 schrieb Patrick Westerhoff:
> Please make sure you???re using the new DownThemAll! version 3 with a current version of AntiContainer. Then everything should work fine. If not, please report back.

Am So den  9. Okt 2016 um 18:18 schrieb Patrick Westerhoff:
> Closed #115.

Yes, works again.

When I created that bug report there was no working version.

Thanks.

Gruß
   Klaus
- -- 
Klaus Ethgen                                       http://www.ethgen.ch/
pub  4096R/4E20AF1C 2011-05-16            Klaus Ethgen <Klaus@Ethgen.ch>
Fingerprint: 85D4 CA42 952C 949B 1753  62B3 79D0 B06F 4E20 AF1C
-----BEGIN PGP SIGNATURE-----
Comment: Charset: ISO-8859-1

iQGcBAEBCgAGBQJX+n2WAAoJEKZ8CrGAGfass4kMAKJR80Iz44MC9PA5Fg4oANUQ
L+F42MtZ3iI74ACsyAo2DQo1D/CAyYFfWHPKTU8eBDpUXoSzJU/lTu5R+ME+KNNk
UAp1mKWCKNJiDgJRWkJt7xNoJ9SBhNlb8Y8JDKwOSG+rhKw50RSb6dAMA4lsr0tH
S9/j3F4xwgutwMnB1fwbw7uJAx88KQ/etjuZaPJ5PgTBGUOTsn6K8r4l6DJptqUU
5Zyzmc2ZKp1mp303WHkgxwdsfPbE9Yne1gONsuSDoGI0ctp2n5kqOAbafFRCiK1R
u14A/KRHFIlKwTSTbDa5g74finqGdvuefRUEfmzu/EzFtRWt/CFng2udZXwCfrOJ
FQhWBHUMc+QjvZ2rTOouQo4YKuwo6A3Lv5l/cPGDh92KaZBCscOeFWk59f8bMQvR
HvStolv6dUbuNPyx84i/8jGz07swuuqStMUOXlIjjEW24kPOZX2T9xLpC2zw9z/V
660n2A3JUrsG5c6TUijfdUS72cUerOtHAlK63QU/WA==
=Qwww
-----END PGP SIGNATURE-----
